### PR TITLE
Changes for Table of Contents

### PR DESCRIPTION
**Pull request by jbydeley-d2l**
There are a few issues that will be resolved in January. _Please_ check out the changes and try it out locally as there is a lot going on here. Let me know if you find any issues.

### CODE REVIEW

File: components/d2l-inner-module.html [hunk 1]
```diff
@@ -30,23 +26,23 @@
 				--d2l-inner-module-border-color: var(--d2l-asv-border-color);
 			}
 
-			.d2l-asv-current {
```
[the remark this answers is not in the document]
Comment (jbydeley-d2l) [reply components/d2l-inner-module.html]: It seems to be working for me? 

image

[the remark this answers is not in the document]
Comment (lawong-d2l) [reply components/d2l-inner-module.html]: @jbydeley-d2l oh I meant, when the module is active itself. Looks like locally inner-seems to have an active effect but not the outer ones.
Something like these: 
https://d2l.invisionapp.com/share/7PP3J7FQ5D9#/screens/331143184

image

[the remark this answers is not in the document]
Comment (lawong-d2l) [reply components/d2l-inner-module.html]: Ignore the yellow wash over that screen shot..... something is clearly wrong with my screen shot program

[the remark this answers is not in the document]
Comment (jbydeley-d2l) [reply components/d2l-inner-module.html]: Let's chat after lunch. Maybe I'm doing something wrong.

[the remark this answers is not in the document]
Comment (jbydeley-d2l) [reply components/d2l-inner-module.html]: nvm, found it 👍

**Pull request merged**